src/remote_worker.c

```diff
@@ -6,23 +6,45 @@
  * "LICENSE" at the root of this distribution.
  */
 
+#include <getopt.h>
 #include "common.h"
 #include "constants.h"
 #include "dcurl.h"
 #include "remote_common.h"
 
-int main(int argc, char const *const *argv)
+int main(int argc, char *const *argv)
 {
     char trytes[TRANSACTION_TRYTES_LENGTH];
     char buf[4];
     int mwm;
+    int cmdOpt;
+    int optIdx;
+    const struct option longOpt[] = {{"broker", required_argument, NULL, 'b'},
+                                     {NULL, 0, NULL, 0}};
 
     amqp_connection_state_t conn;
     amqp_envelope_t envelope;
+    char *hostIP = NULL;
+
+    /* Parse the command line options */
+    /* TODO: Support Mac OS X since getopt_long() is GNU extension */
```

Comment:
Change from "Mac OS X" to "macOS" since Apple already rebrands.

Reply:
You are right.
I have changed it to `macOS`.